dispatch/apps/events/views.py

```diff
@@ -32,6 +35,16 @@ def submit_form(request):
             form.is_submission = True
             form.save()
 
+            template = render_to_string('events/email/confirm.html', form.cleaned_data)
+
+            send_mail(
+                    'Your event has been created!',
+                    template,
+                    settings.EMAIL_HOST_USER,
+                    [form.cleaned_data['submitter_email']],
+                    fail_silently=True,
+                )
```

Comment:
So that would be when the event is approved (and not yet published)? I think that makes more sense, I can implement that